config/locales/en.yml

```diff
@@ -202,3 +202,10 @@ en:
         employment: "Does the claimant’s previous and current schools match the above information from their claim?"
         student_loan_amount: "Does the claimant’s student loan amount and plan type match the information we hold about their loan?"
         matching_details: "Is this claim still valid despite having matching details with other claims?"
+  early_career_payments:
+    policy_name: "Claim a early career payment"
+    policy_short_name: "Early Career Payments"
+    claim_description: "<% #TODO ECP-527 get the correct wording %> ECP-527 GET THE CORRECT WORDING"
```

Comment:
Really want to avoid merging TODOs. They're fine for WIP but not merging into master.

Reply:
@nomasprime Have updated with an educated guess and put in the ticket for James to review.